README.md

```diff
@@ -157,6 +174,10 @@ rarefaction depth of 1500**
 **Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs**  
 `sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
 
+**Calculate the proportion of each source in each sink, using ipyparallel to run in parallel with 5 jobs. Write the full output.**  
+`sourcetracker2 gibbs -i otu_table.biom -m map.txt -o example6/ --jobs 5`
```

Comment:
Fixed these three. I added more description as part of a significant update of the `README.md`.